mne/gui/_coreg_gui.py

```diff
@@ -1194,12 +1212,12 @@ class CoregFrame(HasTraits):
     guess_mri_subject = DelegatesTo('model')
 
     # Omit Points
-    distance = Float(5., label="Distance [mm]", desc="Maximal distance for "
-                     "head shape points from MRI in mm")
-    omit_points = Button(label='Omit Points', desc="Omit head shape points "
+    distance = Float(5., desc="Maximal distance for head shape points from "
+                     "MRI in mm")
+    omit_points = Button(label='Omit [mm]', desc="Omit head shape points "
```

Comment:
As above regarding capitalization of `desc` (I know it's weird for a button...) -- not sure if it can be avoided...